testing/src/main/java/io/grpc/testing/TestUtils.java

```diff
@@ -62,6 +65,7 @@
  * Common utility functions useful for writing tests.
  */
 public class TestUtils {
+  public static final String TESTCA_HOST = "foo.test.google.fr";
```

Comment:
This isn't the test CA host, but more of test server host. Before it didn't matter too much, but this change would make it part of our API, so we would want it to be named accurately.